autode/species/species.py

```diff
@@ -496,6 +496,29 @@ def enthalpy(self) -> Optional[val.Enthalpy]:
             logger.warning('Could not calculate H - an energy was None')
             return None
 
+    @property
+    def zpe(self) -> Optional[val.Energy]:
+        """
+        Zero point vibrational energy of this species
+
+        -----------------------------------------------------------------------
+        Returns:
+            (autode.values.Energy | None): ZPE if frequencies are defined
+        """
+        if self.n_atoms < 2:
+            # A single atom does not have any vibrational energy
+            return val.Energy(0.0)
+
+        if self.vib_frequencies is None:
+            logger.warning('Vibrational frequencies not available, cannot '
+                           'determine zero point energy')
+            return None
+
+        h = 6.62607004E-34  # Planks constant
```

Comment:
don't we have a constants file?

Reply:
We do ([here](https://github.com/duartegroup/autodE/blob/master/autode/constants.py)) but it's full of conversion factors rather than constants with units, currently